.github/workflows/deploy-developer-docs.yml

```
@@ -0,0 +1,52 @@
+name: Deploy on Developer Docs
+
+# Run the deployment only when code is committed to teh branch.
+on:
+  push:
+    branches:
+      # Remove "- developer-doc-github-action" once implemented and keep just `master` branch.
+      - master
+      - developer-doc-github-action
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+
+    - uses: actions/checkout@v2
+
+    - name: Get npm cache directory
+      id: npm-cache
+      run: echo "::set-output name=dir::$(npm config get cache)"
+
+    - name: Configure npm cache
+      uses: actions/cache@v2
+      with:
+        path: ${{ steps.npm-cache.outputs.dir }}
+        key: ${{ runner.os }}-npm-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-npm-
+
+    - name: Build
+      run: npm install && grunt release-no-clean
+
+    - name: Rsync Deployment
+      uses: burnett01/rsync-deployments@4.1
+      with:
+        switches: -avzr --delete
+        path: astra/*
```

```suggestion
        path: astra/
```

rsync does not need this. removing this will get the `--delete` working as expected.

---

@Nikschavan Should we update the same for build and deploy action?

---

Yup we should update it there as well

---

Done